package.json

```diff
@@ -54,42 +74,48 @@
           "default": false,
           "order": 2
         },
+        "codemaker.enableAssistant": {
+          "description": "Enable assistant.",
+          "type": "boolean",
+          "default": false,
+          "order": 3
+        },
```

Comment:
I added one extra setting to InteliJ, which is to control the assistant code completions and opt out of them, the settings will result in disabling the source context contextual operations revering to a regular chat experience.

I am thinking whether that is needed or not, but if the user experience would become degraded by to aggressive attempts to match the code user could always op out of them.

Reply:
Will check. by regular chat experience do you mean AssistantChatCompletion? Which doesn't include any code, only message?